### ISSUE TITLE
Next Screen/Previous Screen does not work in Mac Sierra (version 10.12.6)

### ISSUE THREAD

**Issue by zhujinghui** [fikovnik/ShiftIt#252]
Next Screen/Previous Screen does not work in Mac Sierra (version 10.12.6)

**Comment by waihon** [comment]
I have the same macOS version as @zhujinghui and I encountered the same issue of not being able to move the Next Screen and Previous using both hotkeys as well as selecting from Shiftlt menu.

**Comment by fikovnik** [comment]
I currently do not have apple computer - waiting for a new laptop. Before I had 2009 macbook which did not support Sierra so I have not seen that issue.

**Comment by Ken-B** [comment]
Same on MBP 2015 running osx 10.13.2 (High Sierra). If I enable debugging nothing gets written (I'm probably not doing it right). To be honest, I've just found out about the possibility for next/previous screen and I have not gotten it to work, so I can't say if it's a regression.
Let me know what I can do to help.

Also, I've added a $10 [bountysource](https://www.bountysource.com/issues/49955456-next-screen-previous-screen-does-not-work-in-mac-sierra-version-10-12-6), which I know might not be enough to solve this issue, but it might get things going and it's a small token of my appreciation. Thanks!

**Comment by jlvivero** [comment]
Is there anything that can be done to have the feature to switch an app to a different workspace?

It's a feature that anyone that comes from Linux would probably want, and some paid applications have it (so it should be possible). Is this something in the works?

**Comment by fikovnik** [comment]
@jlvivero - yes, it can be implemented - would you be willing to give it a try - I can provide some pointers.

**Comment by jlvivero** [comment]
yeah I'd be interested, It might take a while since I don't know a lot of objective-C but it's worth a shot!
